ntoskrnl/ex/dbgctrl.c

```diff
@@ -61,11 +61,12 @@ ExpDebuggerWorker(IN PVOID Context)
 
     UNREFERENCED_PARAMETER(Context);
 
-    /* Be sure we were started in an initialized state */
-    ASSERTMSG("ExpDebuggerWorker being entered in non-initialized state!\n",
-              ExpDebuggerWork == WinKdWorkerInitialized);
     if (ExpDebuggerWork != WinKdWorkerInitialized)
     {
+        /* Be sure we were started in an initialized state */
+        ASSERTMSG("ExpDebuggerWorker being entered in non-initialized state! :: ",
+                  ExpDebuggerWork == WinKdWorkerInitialized);
```

Comment:
Please move this assert back above the "if" (otherwise it's just pointless).

Reply:
Moved back.